search/engine.go

```diff
@@ -418,30 +515,49 @@ func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, fro
 		return nil, errors.Wrap(err, "ES error.")
 	}
 
-	if len(mr.Responses) != 2*len(query.LanguageOrder) {
+	if len(mr.Responses) != indexesCount*len(query.LanguageOrder) {
```

Comment:
Here you should use slice of added search requests and use it's length.

Reply:
Issue created AS-36